### ISSUE TITLE
"cci task run create_package_version" should allow option "version_base: default" in cumulusci.yml

### ISSUE THREAD

**Issue by RupertBarrow** [SFDO-Tooling/CumulusCI#3441]
### Feature Type

I'm interested in functionality that solves a current problem or limitation in how CumulusCI works.

### Does your feature request solve a current problem?

Currently, the release_unlocked_beta flow sets option create_package_version__version_base to latest_github_release by default in CCI's code https://github.com/SFDO-Tooling/CumulusCI/blob/6368154b2443db93ae6787a491492e5c6b23ceaa/cumulusci/cumulusci.yml#L1173.

If I want to customize this behaviour to restore the default behaviour to "create_package_version__version_base = the highest existing version number of this package", there is no available syntax to do that.

### Describe the solution you'd like

One should be able to reset that option to default by something like "create_package_version__version_base  default" or ""create_package_version__version_base highest"

### Describe alternatives you've considered

There are none, other than cloning the whole definition of the release_unlocked_beta flow into a custom flow in which create_package_version__version_base would simply not be set

**Comment by yippie** [comment]
I agree this type of change is an issue. Some tasks the only way to specify a specific behavior is to specify nothing/null value for the parameter. However, CumulusCI has no way to override a value with a null/unset.

In general, we need 1 of 2 solutions (or both!)

1) Add support for "unset" overrides when overriding task parameters via CLI options or via flow overrides.
2) Require?/Add explicit values for ALL behavior options. Null for an attribute should set a default value, not EVER be a value in and of itself. There should always be an explicit value available to get the same behavior as not specifying the attribute.
